platforms/chibios/gpio.h

```diff
@@ -41,10 +43,10 @@ typedef uint16_t port_data_t;
 
 #define readPort(pin) palReadPort(PAL_PORT(pin))
 
-#define setPortBitInput(pin, bit) palSetPadMode(PAL_PORT(pin), bit, PAL_MODE_INPUT)
-#define setPortBitInputHigh(pin, bit) palSetPadMode(PAL_PORT(pin), bit, PAL_MODE_INPUT_PULLUP)
-#define setPortBitInputLow(pin, bit) palSetPadMode(PAL_PORT(pin), bit, PAL_MODE_INPUT_PULLDOWN)
-#define setPortBitOutput(pin, bit) palSetPadMode(PAL_PORT(pin), bit, PAL_MODE_OUTPUT_PUSHPULL)
+#define setPortBitInput(pin, bit) palSetPadMode(PAL_PORT(pin), (bit), PAL_MODE_INPUT)
+#define setPortBitInputHigh(pin, bit) palSetPadMode(PAL_PORT(pin), (bit), PAL_MODE_INPUT_PULLUP)
+#define setPortBitInputLow(pin, bit) palSetPadMode(PAL_PORT(pin), (bit), PAL_MODE_INPUT_PULLDOWN)
+#define setPortBitOutput(pin, bit) palSetPadMode(PAL_PORT(pin), (bit), PAL_MODE_OUTPUT_PUSHPULL)
```

Comment:
Hmm, looks like the implementation of `setPortBitOutputPushPull()` and `setPortBitOutputOpenDrain()` is missing here?

Reply:
As discussed on Discord, given that it's not in use anywhere but one keyboard, and it has its own implementation of the port-based GPIO functions, the port-related operations have been removed for the time being until we work out a proper solution for them.